pkg/Library/Fields/MultilineTextField.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * @license
+ * Copyright (c) 2022 Google LLC All rights reserved.
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file.
+ */
+({
+update(inputs, state) {
+  let {value, connectedValue, connectedValue1} = inputs;
```

Comment:
i don't see `connectedValue` or `connectedValue` in the node declaration..

Reply:
Sadly, I think there are two MultilineTextFieldNode.js files?

We gotta sort out all the nodes ASAP.